editor/import/resource_importer_obj.cpp

```diff
@@ -168,18 +168,23 @@ Error ResourceImporterOBJ::import(const String &p_source_file, const String &p_s
 
 					if (face[idx].size() == 3) {
 						int norm = face[idx][2].to_int() - 1;
+						if (norm < 0)
+							norm += normals.size() + 1;
```

Comment:
Not sure about the `+1` on that line (and the others too)
If `norm` is `-1`, then `norm + normals.size() + 1` would be `normals.size()`, which would then fail on the next line...
Are you sure that you don't want to drop the `+ 1`?